CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## dbt-core 1.0.4 (TBD)
+## dbt-core 1.0.5 (TBD)
+- Fixed capitalization in UI for exposures of `type: ml` ([#256](https://github.com/dbt-labs/dbt-docs/issues/256))
```

Comment:
Tiny ask - Change this heading to `## dbt-core 1.1.0 (TBD)`.

Reply:
@emmyoop, thank you for the review. Fixed in f696cf8